<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易计算器</string>
  </property>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_point">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sum">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_muti">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_done">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>41</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_output">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>41</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>导
出
历
史
记
录</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result_history">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>历史记录会出现在这里</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
